static/js/components/VideoCard.js

```diff
@@ -51,13 +50,15 @@ const VideoCard = (props: VideoCardProps) => {
     </div>
     <div className="video-card-body">
       <h4 className="mdc-typography--subheading2">
-        { title }
+        <a href={videoUrl}>{props.video.title}</a>
       </h4>
     </div>
     <div className="actions">
       {
-        props.isAdmin &&
-        <a className="material-icons edit-link" onClick={props.showEditDialog}>mode_edit</a>
+        props.isAdmin && [
+          <a key="delete" className="material-icons delete-link" onClick={props.showDeleteDialog}>delete</a>,
```

Comment:
At certain browser widths the buttons overlay the video title.  The last couple screenshots by @roberthouse54 on #202 shows a single button that when clicked(?) shows 3 options: Edit, Share, Delete.  Is that for a future PR?
![screen shot 2017-09-29 at 9 34 29 am](https://user-images.githubusercontent.com/187676/31019813-f92fe0a0-a4fe-11e7-8463-d9a6450cf46a.png)